src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -740,11 +692,51 @@ public String getStatus() {
         return "";
     }
 
+    // Tracker
```

Comment:
This comment was misplaced during the shuffle

Reply:
Resolved in 28156240aa91125dfeeb163cd5eb6d71869b762a